packages/primitives/table/src/Table.tsx

```diff
@@ -34,6 +33,7 @@ export const Table = component(
       borderLeftWidth: `${borderLeftWidth}px`,
       borderRightWidth: `${borderRightWidth}px`,
       borderStyle,
+      tableLayout: 'fixed',
```

Comment:
but it's already `fixed` in `master` 🤔

Reply:
<img width="868" alt="image" src="https://user-images.githubusercontent.com/13185981/62462806-984b0780-b788-11e9-93fb-256334911f8c.png">
Somebody was faster 😄